src/modules/modals/CreateRow.vue

```diff
@@ -151,6 +156,38 @@ export default {
 </script>
 <style lang="scss" scoped>
 
+.modal__content {
+	padding: 20px;
+
+	:where(.row .space-T, .row.space-T) {
```

Comment:
I think we don't need the `:where` here as with scss those would automatically be converted to `.modal__content .row .space-T, .modal__content .row.space-T`